managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx

```diff
@@ -0,0 +1,464 @@
+</Message>
+
+### Using Meltano
+
+Tools such as [Meltano](https://meltano.com/) are better suited to load data only, without keeping constraints on column types or relations. To load all PostgreSQL objects (such as table constraints, users, etc.), we recommend using other tools such as [pg_dump and pg_restore](#using-pgdump).
+
+#### Prerequisites
+
+To complete the following procedure, you must have:
+ 
+- Installed [Meltano](https://docs.meltano.com/getting-started/installation/)
+
+#### Initializing your Meltano project
+
+1. In a terminal, run the command below to initialize a Meltano project named `migrate-postgresql`:
+
+    ```sh
+    meltano init migrate-postgresql
+    ```
+
+2. Run the following command to access the newly created directory that contains the project:
+
+    ```sh
+    cd migrate-postgresql
+    ```
+
+3. Run the following command to add a PostgreSQL-compatible extractor and loader:
+
+    ```sh
+    meltano add extractor tap-postgres
+    meltano add loader target-postgres
+    ```
+
+#### Configuring the extractor to connect to your existing database
+
+1. Run the command below to configure the connection to your existing database:
+
+    ```sh
+    meltano config tap-postgres set --interactive
+    ```
+
+    A list of settings displays, with a number associated with each parameter.
+
+2. Fill in the required information below by entering the number associated with each parameter:
+
+    - **Database Name** (5)
+    - **Host** (13)
+    - **Password** (14)
+    - **Port** (15)
+    - **Username** (32)
+
+3. Select the tables you want to load:
+
+    ```
+    meltano select tap-postgres {schema-name}-{table1name}
+    meltano select tap-postgres {schema-name}-{table2name}
+    ...
+    ```
+
+    <Message type="tip">
+    For instance `meltano select tap-postgres public-cities` if you are importing the table `cities` from the schema `public`.
+    You can check the selected tables and columns with `meltano select tap-postgres --list --all`.
+    </Message>
+
+#### Configuring the loader to connect to your new Managed Database for PostgreSQL
+
+1. Run the command below to configure the connection to your existing database:
+
+    ```sh
+    meltano config tap-postgres set --interactive
+    ```
+
+    A list of settings displays, with a number associated with each parameter.
+
+2. Fill in the required information below by entering the number associated with each parameter:
+
+    - **Database Name** (5)
+    - **Host** (13)
+    - **Password** (14)
+    - **Port** (15)
+    - **Username** (32)
+
+    <Message type="note">
+    You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database. The password is the [IAM secret key](/identity-and-access-management/iam/concepts/#api-key) corresponding to the [IAM user](/identity-and-access-management/iam/concepts/#user) or [IAM application](/identity-and-access-management/iam/concepts/#application) you are connecting with. 
```

Comment:
Managed PostgreSQL is not integrated with IAM yet so we need to replace this with "the username & password that you defined upon Database Instance Creation"